python/probe_screen.py

```diff
@@ -2015,10 +2059,9 @@ def on_spbtn_block_height_value_changed( self, gtkspinbutton, data = None ):
             self.warning_dialog( self, _( "Conversion error in btn_block_height!" ),
                                    _( "Please enter only numerical values\nValues have not been applied" ) )
         # set koordinate system to new origin
-        origin = float(self.inifile.find("AXIS_2", "MIN_LIMIT")) + blockheight
         self.command.mode( linuxcnc.MODE_MDI )
         self.command.wait_complete()
-        self.command.mdi( "G10 L2 P0 Z%s" % origin )
+        self.command.mdi( "G10 L2 P0 Z%s" % blockheight )
```

Comment:
Again, not strictly a 2.8 change. However, without this change - enabling and using M6 remap would always result in the Z offset being off by the Z Axis's MIN_LIMIT.